project.rosinstall

```diff
@@ -6,7 +6,7 @@
 - git:
     local-name: universal_robot
     uri: 'https://github.com/shadow-robot/universal_robot.git'
-    version: F_sr_plugin_compatible_latest
+    version: F_sr_plugin_compatible
```

Comment:
OK. maybe we should either change this to F_trajectory_record_play_stl_collision or indicate in the docs that the branch F_sr_plugin_compatible should not be used with a real arm (only simu).